### DISCUSSION TITLE
Documentation Help

### DISCUSSION THREAD

**Discussion by minpark283** [milvus-io/milvus#31415] [Q&A and General discussion] [answered]
Hello,
I am trying to use pymilvus, but I am having difficult in deciding which version of pymilvus to use, and which documentation I should be looking at.

My main concern is that, I see in this documentation:
https://milvus.io/api-reference/pymilvus/v2.3.x/About.md
that all documentation/features under PyMilvus is a legacy and is deprecated, while MilvusClient is the one being supported now.
But I see that in https://milvus.io/docs under userguide, the examples are still with the legacy stuff.

Would it be correct to use the features and documentation under MilvusClient instead of PyMilvus?
As of right now, would the latest Milvus 2.3.x be the best version to use in terms of stability and being well-documented?

I apologize in advance if this is something that was already asked, but I couldn't find a similar discussion on this.

**Comment by yhmo** [comment, marked as answer]
MilvusClient is a new class to organize the interfaces for usability, it is a part of pymilvus. The underlying RPC interfaces are the same. The old methods are marked as legacy, however they will be continually supported.

For new users, MilvusClient is recommended to use. For old users, no need to replace the legacy script by MilvusClient, the legacy script also works.

Typically, the latest version is the most stable version. All the documents are updated according to the latest version.
